SandboxUwp/MainPage.xaml.cs

```diff
@@ -72,6 +72,32 @@ public MainPage()
             OutputBlock.Text = "Created blood pressure.";
         }
 
+        private async void Get_Height_OnClick(object sender, RoutedEventArgs e)
+        {
+            PersonInfo personInfo = await connection.GetPersonInfoAsync();
+            HealthRecordInfo recordInfo = personInfo.SelectedRecord;
+            IThingClient thingClient = connection.CreateThingClient();
+            var heights = await thingClient.GetThingsAsync<Height>(recordInfo.Id);
+            Height firstHeight = heights.FirstOrDefault();
+            if (firstHeight == null)
+            {
+                OutputBlock.Text = "No height.";
+            }
+            else
+            {
+                OutputBlock.Text = firstHeight.Value.Meters.ToString();
```

Comment:
This could use some simple formatting, so that there's something other than just the value.  (e.g., `1.8m` instead of just `1.8`)